incubator/artifactory/values.yaml

```diff
@@ -8,7 +8,7 @@ Component: "Artifactory"
 
 ## Uncomment following line if you want to run Artifactory-Pro
 #image: "docker.bintray.io/jfrog/artifactory-pro"
```

Comment:
Since we're dropping the `default` statements, set the image to `docker.bintray.io/jfrog/artifactory-oss`.